src/Servers/Kestrel/test/InMemory.FunctionalTests/Http3/Http3ConnectionTests.cs

```diff
@@ -73,6 +74,58 @@ public async Task CreateRequestStream_RequestCompleted_Disposed()
             Assert.True(requestStream.Disposed);
         }
 
+        [Fact]
+        public async Task HEADERS_Received_ContainsExpect100Continue_100ContinueSent()
+        {
+            var appCompletedTcs = new TaskCompletionSource(TaskCreationOptions.RunContinuationsAsynchronously);
+            await Http3Api.InitializeConnectionAsync(async context =>
+            {
+                var buffer = new byte[16 * 1024];
+                var received = 0;
+
+                while ((received = await context.Request.Body.ReadAsync(buffer, 0, buffer.Length)) > 0)
+                {
+                    await context.Response.Body.WriteAsync(buffer, 0, received);
+                }
+
+                await appCompletedTcs.Task;
```

Comment:
Unused, remove.

Reply:
Yes, get rid of the TCS. Also, you can add `await requestStream.ExpectReceiveEndOfStream();` to the end of the test to verify the request's response stream ended correctly.